src/tuning/kernels/xgemm.cpp

```diff
@@ -63,7 +63,7 @@ class TuneXgemm {
   static double DefaultInfluenceGlobalPSO(){ return 0.1; }
   static double DefaultInfluenceLocalPSO(){ return 0.3; }
   static double DefaultInfluenceRandomPSO(){ return 0.6; }
-  static size_t DefaultHeuristic(){ return size_t{3};} // PSO
+  static size_t DefaultHeuristic(){ return (size_t)cltune::SearchMethod::PSO;} // PSO
```

Comment:
Good idea to use the `cltune::SearchMethod` type! Now that we have that, can't we make the return type of this to be of that type as well (get rid of the `size_t`)? And then that can propagate further and can be directly set, right?

P.S. The comment is no longer needed here :-)